src/sass/components/_buttons.scss

```diff
@@ -81,6 +80,7 @@ Styleguide 2.0
 
   &:link{
     color: $link;
+    text-decoration: underline;
```

Comment:
I dig it, I think we can leave it as just removing the text-decoration line as you did above without adding this one explicitly here :)